test/specs/xsrf.spec.js

```diff
@@ -28,6 +28,19 @@ describe('xsrf', function () {
     });
   });
 
+  it('should not set xsrf header if xsrfCookieName is null', function (done) {
```

Comment:
I think we can make an assumption that the adapter doesn't access `document.cookie` directly, but rather uses the `cookies` helper. It's seems to be much easier to spy on a function than on a property with Jasmine.